libs/blocks/quiz/quiz.js

```diff
@@ -202,6 +210,15 @@ const App = () => {
   const minSelections = +selectedQuestion['min-selections'];
   const maxSelections = +selectedQuestion['max-selections'];
 
+  const fragmentURL = getStringValue('footerFragment');
+  
+
+  useEffect(() => {
```

Comment:
Talking with @JackySun9, it seems like Milo's createFragment method is pretty heavy. Perhaps it is worthwhile to set the footerFragment value in strings.json to a text string if it's all we need. We can use a fragment URL for the modal if needed, but otherwise, maybe we can get away cheaply? What if we did something like the following?

  const footerFragment = getStringValue('footerFragment');

  useEffect(() => {
    if (footerFragment.startsWith("https://")) {
      loadFragments(footerFragment);
    }
    else {
    	const quizFooter = document.querySelector('.quiz-footer');
		quizFooter.innerHTML = footerFragment;
    }
  }, [footerFragment]);

Reply:
I with draw my comment in light of your new evidence. Thanks!

Reply:
@echampio-at-adobe  @JackySun9 
was able to find the issue as soon step-indicator was not the last child the css was not loaded 

for instance just moved the quiz-footer to be before the indicator